firestore/tests/unit/v1/test_transaction.py

```diff
@@ -329,6 +329,109 @@ def test__commit_failure(self):
             metadata=client._rpc_metadata,
         )
 
+    def _get_all_helper(self, client, document_pbs):
+        # Create a minimal fake GAPIC with a dummy response.
+        firestore_api = mock.Mock(spec=["batch_get_documents"])
+        response_iterator = iter(document_pbs)
+        firestore_api.batch_get_documents.return_value = response_iterator
+
+        # Attach the fake GAPIC to a real client.
+        client._firestore_api_internal = firestore_api
+
+    def test_get_all(self):
+        from google.cloud.firestore_v1.document import DocumentSnapshot
+        import types
+
+        client = _make_client()
+        trans = self._make_one(client)
+        data1 = {"a": u"cheese"}
+        data2 = {"b": True, "c": 18}
+        document1 = client.document("pineapple", "lamp1")
+        document2 = client.document("pineapple", "lamp2")
+        document_pb1, read_time = _doc_get_info(document1._document_path, data1)
+        document_pb2, read_time = _doc_get_info(document2._document_path, data2)
+        response1 = _make_batch_response(found=document_pb1, read_time=read_time)
+        response2 = _make_batch_response(found=document_pb2, read_time=read_time)
+
+        self._get_all_helper(client, [response1, response2])
+        # Actually call get_all().
+        snapshots = trans.get_all([document1, document2])
+        self.assertIsInstance(snapshots, types.GeneratorType)
+        snapshots = list(snapshots)
+        snapshot1 = snapshots[0]
```

Comment:
I'd rather name it `snapshot0 = snapshots[0]`, `snapshot1 = snapshots[1]` etc., to make it more intuitive.

Reply:
ok, I will do that if you want, but the same thing "snapshot1 = snapshots[0]" like is in the test_client.py file in the same method def test_get_all(), so  to uniform everything I use it,please verify it, if you want  i will change that variable name

Reply:
This is not critical, was merely a suggestion. You can do it either way.